services/graph/pkg/server/http/option.go

```diff
@@ -22,6 +23,7 @@ type Options struct {
 	Flags         []cli.Flag
 	Namespace     string
 	TraceProvider trace.TracerProvider
+	Store         microstore.Store
```

Comment:
This store option is passed through several layer of the code, but I don't think it is used anywhere.